src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -717,7 +717,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    */
   _disableTextTracks(): void {
     let textTracks = this._videoElement.textTracks;
-    if (textTracks) {
+    if (textTracks && textTracks.kind !="metadata") {
```

Comment:
should be `["captions", "subtitles"].includes(textTracks.kind);` as there are [other kinds as well](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track).
And also, we should add textrack kinds as an ENUM in the `./tracks` folder and import it here so we can use it as
```javascript
import {TEXT_TRACK_KINDS} from ...
TEXT_TRACK_KINDS.CAPTIONS;
TEXT_TRACK_KINDS.SUBTITLES;
```

so we can avoid some previously notorious bugs with caption vs captions string errors....

Reply:
do we need to handle this in hls and dash adapters as well?

Reply:
Since both are using the video tag - this cover all of them.